src/api/object_api.js

```diff
@@ -535,6 +535,33 @@ module.exports = {
             }
         },
 
+        read_upload_md: {
+            method: 'GET',
+            params: {
+                type: 'object',
+                required: [
+                    'bucket',
+                    'key',
+                ],
+                properties: {
+                    bucket: { $ref: 'common_api#/definitions/bucket_name' },
+                    key: { type: 'string' },
+                }
+            },
+            reply: {
+                type: 'object',
+                properties: {
+                    obj_id: { objectid: true },
+                    key: { type: 'string' },
+                    xattr: { $ref: '#/definitions/xattr' }
+                }
+            },
```

Comment:
Wait, what is this api about? returning an upload from key? but there could be multiple uploads per key.
Can you elaborate a little on the case that requires it and how to handle concurrent/leftover uploads to the same key..

Reply:
deleted from api